maat-court-data-api/src/test/java/gov/uk/courtdata/billing/repository/ApplicantResetRepositoryTest.java

```diff
@@ -0,0 +1,58 @@
+package gov.uk.courtdata.billing.repository;
+
+import gov.uk.courtdata.billing.entity.ApplicantEntity;
+import jakarta.persistence.EntityManager;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest
+@Sql(statements = {
+        "CREATE SCHEMA IF NOT EXISTS TOGDATA;",
+        """
+        CREATE TABLE IF NOT EXISTS TOGDATA.APPLICANTS (
+            ID INT PRIMARY KEY,
+            SEND_TO_CCLF VARCHAR(1),
+            DATE_CREATED TIMESTAMP,
+            USER_CREATED VARCHAR(100),
+            DATE_MODIFIED TIMESTAMP,
+            USER_MODIFIED VARCHAR(100)
+        );
+        """
+})
+class ApplicantResetRepositoryTest {
```

Comment:
Very cool, I wasn't aware we had the capacity to test at the repository level. Nice 💯